.travis.yml

```diff
@@ -4,6 +4,7 @@ cache: npm
 before_script:
   - npm run lint
 before_deploy:
+  - npm install
```

Comment:
Why adding this line? AFAIK, this is automatically done by Travis when working on a node environment.

Reply:
I had some problems when I added the `vue-map` dependencie, but maybe was for don't including the `package-lock.json`. I will remove it and try again without it.